daprdocs/content/en/reference/components-reference/supported-bindings/http.md

```diff
@@ -20,13 +20,23 @@ spec:
   metadata:
   - name: url
     value: http://something.com
+  - name: MTLSRootCA
+    value: /Users/somepath/root.pem #<path to root CA> or <pem encoded string>
+  - name: MTLSClientCert
+    value: /Users/somepath/client.pem #<path to client cert> or <pem encoded string>
+  - name: MTLSClientKey
+    value: /Users/somepath/client.key #<path to client key> or <pem encoded string>
+
 ```
 
 ## Spec metadata fields
 
 | Field              | Required | Binding support | Details | Example |
 |--------------------|:--------:|--------|--------|---------|
 | url                | Y        | Output |The base URL of the HTTP endpoint to invoke | `http://host:port/path`, `http://myservice:8000/customers`
+| MTLSRootCA         | N        | Output |Path to root ca certificate or pem encoded string |
```

Comment:
Can you provide examples for the paths. Always good to clarify if this can be absolute path or relative path and if relative, relative to what.